.github/workflows/helm_charts_scalar.yml

```
@@ -177,6 +177,7 @@ jobs:
           kubectl create secret generic auditor-keys --from-file=certificate=.github/auditor-cert.pem --from-file=private-key=.github/auditor-key.pem
           kubectl create -f .github/cassandra.yaml
           echo "Wait until cassandra is running" && sleep 100
```

```suggestion
          echo "Wait until cassandra is running" && sleep 100
          echo -n "Wait for Cassandra "
          until kubectl exec cassandra-0 -- cqlsh 2> /dev/null; do  sleep 0.5; echo -n "#"; done
```
As I mentioned in another comment, I think we can use this command for waiting for Cassandra to start.

---

It looks like you can use `kubectl wait` command by setting up the readiness probe for cassandra Pod.

```
diff --git a/.github/cassandra.yaml b/.github/cassandra.yaml
index e0a8a67..fc0e677 100644
--- a/.github/cassandra.yaml
+++ b/.github/cassandra.yaml
@@ -21,6 +21,9 @@ spec:
           env:
             - name: CASSANDRA_SEEDS
               value: "cassandra-0.cassandra-headless.default.svc.cluster.local"
+          readinessProbe:
+            exec:
+              command: ["cqlsh", "-e", "SELECT now() FROM system.local;"]
 ---
 apiVersion: v1
 kind: Service
```

```
kubectl wait --for=condition=Ready pod/cassandra-0
```